### ISSUE TITLE
Mixed up isNextSentence label in simple_lm_finetuning.py script? 

### ISSUE THREAD

**Issue by yakazimir** [huggingface/pytorch-pretrained-BERT#524]
I'm wondering if the isNextsentence "label" in the below function is correct? Shouldn't the label be 1 in the case that t1,t2 are taken from self.get_corpus_line(index) (i.e., the first condition on line 150), and 0 if it is random (line 153)? 

https://github.com/huggingface/pytorch-pretrained-BERT/blob/c36cca075a32f59a5ec2083e1d39e7d6564c105b/examples/lm_finetuning/simple_lm_finetuning.py#L141-L157

**Comment by thomwolf** [comment]
Hi, why should it be the other way around?

**Comment by yakazimir** [comment]
I think I mixed up the meaning of 0 and 1 in this context and maybe wrote this post a bit too quickly before looking deeper into the code and documentation.. (sorry!). On second glance, the documentation for the BertForPreTraining is rather clear (https://github.com/huggingface/pytorch-pretrained-BERT/blob/d76a57b0ba198eee27b3777f57fcabb6aba8b965/pytorch_pretrained_bert/modeling.py#L766): 

``` 0 => next sentence is the continuation, 1 => next sentence is a random sentence.``` 

I was confused why 0 should mean "true" is this case (i.e., is a next sentence continuation) since in classification 0 often means "false", but whatever, the way it is written is sound (albeit a little counterintuitive at first glance).

**Comment by askerlee** [comment]
@yakazimir yeah I was confused too. Thanks for your research
